draft-deepspace-ip-architecture.xml

```diff
@@ -218,46 +396,133 @@
+	features suitable and useful in deep space such as 1 RTT for
+	connection establishment and security, mobility, 0 RTT,
+	streams, user space, etc. [TLI: This sentence needs more words
+	to explain these references.]</t>
+        <t>Current implementations of QUIC typically set various
+	transport configuration parameters suitable for the Internet
+	environment, expecting an RTT to be in the hundreds of milliseconds
+	and a normally always-connected network. Therefore, QUIC
+	stacks using default configurations will not work in deep
+	space. However, studies and simulations <xref
+	target="quic-sim"/> showed that with proper configuration of
+	parameters, QUIC stacks can support the delays and disruptions
+	in deep space. <xref
+	target="I-D.many-deepspace-quic-profile"/> describes how to
+	properly configure a QUIC stack for deep space applications,
+	where QUIC is unaware of disruptions. If the transport is
+	aware of the disruptions, further optimizations are possible.</t>
+        <t>Having multiple streams and applications within a single
+	QUIC connection is valuable and useful for deep space. A
+	ground station may set up the initial QUIC connection with a spacecraft and then carry all needed applications and streams over that same connection for the whole duration of the mission.</t>
+        <t>Session keys and certificate lifetimes together with certificate validation and trust chain anchors need to be carefully configured and handled.</t>
+        <t>QUIC proxies <xref target="I-D.ietf-masque-quic-proxy"/>
+	can be used at the edge of space to isolate, apply policies, or
+	optimize traffic at the ingress/egress to a celestial body network.</t>
       </section>
       <section>
         <name>Other Transports</name>
-        <t>Other transports such as TCP<xref target="RFC9293"/>, SCTP<xref target="RFC9260"/>, DCCP<xref target="RFC4340"/> and others were not investigated for their suitability in space.</t>
+        <t>Other transports such as TCP <xref target="RFC9293"/>, SCTP
+	<xref target="RFC9260"/>, DCCP <xref target="RFC4340"/> and others were not investigated for their suitability in space.</t>
       </section>
       <section anchor="ecn">
-        <name>Explicit Congestion Notification(ECN)</name>
-        <t>Explicit Congestion Notification(ECN)<xref target="RFC3168"/> enables a network forwarder to signal to the sources to pace down in the context of congestion. In deep space where forwarders will be buffering packets, the actual congestion is when the storage is approaching its full capacity. ECN enables the forwarders to signal that issue. Given delays and disruptions which means the reception by the source may take longer time, the forwarders should be pro-active and preempt based on the types of links and actual rate of storage usage. IP clients with QUIC should process the received ECN signals appropriately.</t>
+        <name>Explicit Congestion Notification (ECN)</name>
```

Comment:
The editorial changes are fine with me, but I wanted to comment here that the algorithm(s) for setting ECN marks and responding to them in such a deepspace scenario will need to be created.  Algorithms like RED, CoDel, PIE, etc. are all assuming that the sources are responsive.  In this case, however, a source can burst into the queue at multiple Gbps (over a terrestrial network), then packets will can sit in the queue for N hours, and the congestion probably needs to be measured related to the expected transmission rate and duration predicted when the queue starts draining, rather than just the size of the buffer itself (unless that's autotuned based on the transmission opportunities?).